next-docs/public/examples/tabsTW/SegmentControl.tsx

```diff
@@ -0,0 +1,60 @@
+import { Tab } from '@headlessui/react';
+import React, { Fragment, useState } from 'react';
+
+const items = ['$Link', '$Link', '$Link', '$Link'];
+
+const Example = () => {
+  return (
+    <>
+      <Tab.Group>
+        <Tab.List className="flex h-10 p-1 bg-beerus rounded-xl">
```

Comment:
Please, use moon radiuses. in this case it's interactive. Never use TW radiuses and font sizes, where it's possible to use moon ones.